gulpfile.js

```diff
@@ -15,10 +15,19 @@
  */
 
 const gulp = require('gulp');
+const gutil = require('gulp-util');
 const Karma = require('karma').Server;
 const karmaDefault = require('./karma.conf');
 
+
 gulp.task('test', function(done) {
+   try {
+      require('./readiness-tool/vendors.json');
+      console.log("vendors.json was found with no problems");
+    } catch (err){
+      gutil.log(gutil.colors.red('vendors.json failed to load because this error: ' + err.message));
+      return;
+    }
```

Comment:
As discussed, this is because the unit test uses `require` but the source uses `fetch`. I guess this also accomplishes the task although it's not ideal.